### ISSUE TITLE
easynews addon freezes/buffering poor performance

### ISSUE THREAD

**Issue by itzikvil** [sleeyax/stremio-easynews-addon#51]
Hן,

I have an issue with buffering/freezing using Easynews+ addon which I did not suffer from few months ago and its quite frustrating, here is the thing, while working with same addon (1.3.2 now 1.3.4) I always get buffering (freezing) while watching Movies through stremio also when I skip forward to see response or when starting a movie i get about 10-15 seconds delay, this was working smooth before, i have also torrentio and its working smooth 1-3 seconds for both actions fluently. can you please advise? have you changed something within addon which can impact/limit?

thanks a lot,

itzik

**Comment by sleeyax** [comment]
No, I haven't changed anything recently that would impact this. It's difficult for an addon to be able to have such an impact on performance in the first place.

This might be related to a performance issue with Stremio itself. See https://github.com/Stremio/stremio-bugs/issues/984 and #41. I'm working with them in an attempt to improve usenet (easynews) streaming support - which will result in a smoother streaming experience from usenet sources overall - but it might take a while and there's no clear ETA for when this would be implemented.

**Comment by itzikvil** [comment]
thanks, mate you are doing an amazing job. I understand what you are saying and totally agree with you regarding addon modification impact on slowness though I doubt if the problem you responded with is somehow related to the performance i experience within stremio app as it smells to me according to my experience route to easynews closest server or something with pre buffering can't tell exactly, keep on doing what is needed it feels like we are in good hands with this. let me know if that make sense or if you have another idea of how to eliminate this poor performance phenomenon.

**Comment by MrXT21** [comment]
https://members.easynews.com/1.0/route/ 

Run a test and then select the fastest port.

**Comment by itzikvil** [comment]
Hi,

thanks for reply, i already done this several times and it did not help. any more ideas? by the way Easy News support is unresponsive and it is really getting weird from day to day. I email them have an open incident and no reply.

**Comment by MrXT21** [comment]
> Hi,
> 
> thanks for reply, i already done this several times and it did not help. any more ideas? by the way Easy News support is unresponsive and it is really getting weird from day to day. I email them have an open incident and no reply.

If it isnt working for you then i suggest you get a refund before the 14 day period runs out. I know thats probably not what you want to hear but unfortunately if nothings worked and they dont reply it isnt worth paying for.

I have noticed a few times now that streams never start or dont start before i lose patience and try another link and pausing for a longer than a min results in the stream not resuming. Iv really not been impressed with usenet.

**Comment by itzikvil** [comment]
> > Hi,
> > thanks for reply, i already done this several times and it did not help. any more ideas? by the way Easy News support is unresponsive and it is really getting weird from day to day. I email them have an open incident and no reply.
> 
> If it isnt working for you then i suggest you get a refund before the 14 day period runs out. I know thats probably not what you want to hear but unfortunately if nothings worked and they dont reply it isnt worth paying for.
> 
> I have noticed a few times now that streams never start or dont start before i lose patience and try another link and pausing for a longer than a min results in the stream not resuming. Iv really not been impressed with usenet.

Hi MrXT21,

believe me i would have ask for refund but there are few reasons why I am not going for it at the moment.
- I hope things will get better from Addon integration side counting on Developers as i know they are professionals and trying to do their best in contrary to Easynews responsiveness and support which is ridiculous and outrages beyond every expectation. i was very disappointed to say the least.
- I purchased Easynews on black Friday, so it was very low price, 30$ which if I compare it to an Old car, you have which takes you from A to B (which even that easynews don't do at the moment) and if you sell it you will get cheap price so you prefer to have a car that is driving then sell it and get peanuts.
- I trialed easynews with the same addon about 4 months ago from the same LAN i am currently working and its performance was pretty good which shows its not my side and something either on the route/addon/easynews side was changed badly, hence i am hoping it will be fixed and will work like 4 months ago.

Saying all that if i will see it is not getting better (and as i see its not only me who "suffer") then i will surely remove the addon and close the account as its useless for me, especially as there are many and there will be other options to stream data.

too bad for all the hard work developers do that in the end it goes down the drain, any hope i cross hands that they will take it seriously and take care of the problems and make addon work satisfactory as now it is impossible unbearable to work with.
-

**Comment by MrXT21** [comment]
> > > Hi,
> > > thanks for reply, i already done this several times and it did not help. any more ideas? by the way Easy News support is unresponsive and it is really getting weird from day to day. I email them have an open incident and no reply.
> > 
> > 
> > If it isnt working for you then i suggest you get a refund before the 14 day period runs out. I know thats probably not what you want to hear but unfortunately if nothings worked and they dont reply it isnt worth paying for.
> > I have noticed a few times now that streams never start or dont start before i lose patience and try another link and pausing for a longer than a min results in the stream not resuming. Iv really not been impressed with usenet.
> 
> Hi MrXT21,
> 
> believe me i would have ask for refund but there are few reasons why I am not going for it at the moment.
> 
>     * I hope things will get better from Addon integration side counting on Developers as i know they are professionals and trying to do their best in contrary to Easynews responsiveness and support which is ridiculous and outrages beyond every expectation. i was very disappointed to say the least.
> 
>     * I purchased Easynews on black Friday, so it was very low price, 30$ which if I compare it to an Old car, you have which takes you from A to B (which even that easynews don't do at the moment) and if you sell it you will get cheap price so you prefer to have a car that is driving then sell it and get peanuts.
> 
>     * I trialed easynews with the same addon about 4 months ago from the same LAN i am currently working and its performance was pretty good which shows its not my side and something either on the route/addon/easynews side was changed badly, hence i am hoping it will be fixed and will work like 4 months ago.
> 
> 
> Saying all that if i will see it is not getting better (and as i see its not only me who "suffer") then i will surely remove the addon and close the account as its useless for me, especially as there are many and there will be other options to stream data.
> 
> too bad for all the hard work developers do that in the end it goes down the drain, any hope i cross hands that they will take it seriously and take care of the problems and make addon work satisfactory as now it is impossible unbearable to work with.

After using it for a while now i see exactly what you mean. Many streams are taking forever to begin or just eternally load until you give up and try another source. I was trying beverly hills cop 2 the other day and i tried numerous sources and nothing would load at all until i clicked about 8 different sources and then finally a source loaded though that too cut out after about 45mins and i had to back out and click it again.

Some streams play but cut out after an hour or so and this issue is especially prevalent on android TV. I wonder if the addon is reverting to a default port or choosing one at random each time you start a stream because if i back out of a stream and try the same one a few times it suddenly starts almost immediately when before it would load for ages until I back out.
I dont know whats happening here but given the speed of easynews servers and my GB/s internet speed everything should be a hell of alot faster than it is, if anything it should be faster than any debrid server/provider.

**Comment by itzikvil** [comment]
Hi mate,

Saw your comment and i do symphaty.

All you mention and expreienced i already communicated with both developer in this post and easynews non responsive support both say they are 100% ok and send you to find a solution for yourself. I really dont see reason for this attitude since what is the point of building and developing investing so much time for good intention and when you get a problem and whiling from customer to assist in solving it ignoring and leaving it as is or accusing the other service orovider for the glitch.
I am not saying by the way that it is the easynews addon dont missunderstood me high chance that it is the route or inability of the network infra for easynews to provide appropriate service both i gave enough reasonable checkes to isolate the problem and was also commited to cooperate with bith developer and easynews to get a better service from their servers and from the addon.
I git zero cooperation and zero responsivness so i am not bringing it anymore after all developer doing it in their free time and voulentirly but that is the status now and too bad after all time invested in this it is stuck and getting no real remedy.

**Comment by MrXT21** [comment]
> Hi mate,
> 
> Saw your comment and i do symphaty.
> 
> All you mention and expreienced i already communicated with both developer in this post and easynews non responsive support both say they are 100% ok and send you to find a solution for yourself. I really dont see reason for this attitude since what is the point of building and developing investing so much time for good intention and when you get a problem and whiling from customer to assist in solving it ignoring and leaving it as is or accusing the other service orovider for the glitch. I am not saying by the way that it is the easynews addon dont missunderstood me high chance that it is the route or inability of the network infra for easynews to provide appropriate service both i gave enough reasonable checkes to isolate the problem and was also commited to cooperate with bith developer and easynews to get a better service from their servers and from the addon. I git zero cooperation and zero responsivness so i am not bringing it anymore after all developer doing it in their free time and voulentirly but that is the status now and too bad after all time invested in this it is stuck and getting no real remedy.

This may actually be a stremio issue. I recently tried the easynews add-ons on nuvio and it works great. Streams start instantly and don't appear to freeze in my testing. If you still have an active subscription id recommend giving it a go. 
It also works great with aiostreams and stremio new Usenet streaming feature and third party indexers.
